### DISCUSSION TITLE
Publishing an AAB file

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#2808] [General]
Good afternoon. Tell me if there is an instruction for publishing aab files for google play. I use Abobe Animate

**Comment by muriloflesch2** [comment]
Open the adt.cfg file that is in within your Air SDK/LIB/ADT.cfg folder.

UNCONTENDED 2 LINES:
CreateandroidAndPPBundle = True
KeepandroidTudiooutput = True

:)

**Reply by skramovskiy** [reply]
спасибо!)
